doc/tutorials/basic/explore_data.md

```diff
@@ -0,0 +1,194 @@
+# Explore the Data
+
+In this section, we will use the
+
+- [Swipe](../../reference/layouts/Swipe.ipynb) *layout* to understand how the global mean surface temperature has risen from 1945 to 2015
+- [Perspective](../../reference/panes/Perspective.ipynb) *pane* to slice and dice the wind `turbines` dataset.
+- [Vega](../../reference/panes/Vega.ipynb) (Altair) *pane*, the [`Player`](../../reference/widgets/Player.ipynb) *widget* and `pn.bind` to understand how manufacturer capacity has developed since the 80's via a *bar chart race*.
+
+This will barely scratch the surface of what Panel has to offer for an exploratory data analysis (EDA) workflow. That is why we say *Panel is **the powerful data exploration** and web app **framework***.
+
+:::{note}
+When we ask to *run the code* in the sections below, we may either execute the code directly in the Panel docs via the green *run* button, in a cell in a notebook, or in a file `app.py` that is served with `panel serve app.py --autoreload`.
+:::
+
+```{pyodide}
```

Comment:
Is it possible to hide this code in the docs @philippjfr ? Its there for technical reasons. We must import all javascript at the top of the document. But users can be confused by this.